src/components/abstractComponent.ts

```diff
@@ -226,7 +223,7 @@ export module Component {
      * @param {number} [availableHeight] - the height of the container element
      * @returns {Component} The calling component.
      */
-    public resize(width?: number, height?: number): AbstractComponent {
+    public forceRedraw(width?: number, height?: number): AbstractComponent {
```

Comment:
I was thinking `forceRedraw()` wouldn't take arguments. Plottable infers the size of the `<svg>`, so it shouldn't be necessary to tell it anything. It might make sense to have `resize(width, height)` remain as a separate API point (with the arguments now mandatory).